daprdocs/data/components/pubsub/generic.yaml

```diff
@@ -1,6 +1,6 @@
 - component: Hazelcast
   link: setup-hazelcast
-  state: Beta
+  state: Deprecated
   version: v1
   since: "1.7"
```

Comment:
Since 1.9 or 1.10?

Reply:
The log message will be added in 1.10, but we can say that the component is considered deprecated as of today, regardless of the version (no point in people starting to use Hazelcast now if they were not using it before)

Reply:
Make the since version either 1.9 or 1.10 to reflect from when the component is deprecated

Reply:
Changed